cedar-policy/src/tests.rs

```diff
@@ -3258,3 +3258,56 @@ mod error_source_tests {
         }
     }
 }
+
+mod issue_604 {
+    use crate::Policy;
+    fn to_json_is_ok(text: &str) {
+        let policy = Policy::parse(None, text).unwrap();
+        let json = policy.to_json();
+        assert!(json.is_ok());
+    }
+
+    #[test]
+    fn var_as_attribute_name() {
```

Comment:
are there other attribute names we should test? `has`, `is`, `if`, `permit`, `else`, etc?  Some of these would be actually errors, but we could add assertions that they're errors, just so the tests let us know if the behavior ever changes

Reply:
Yes. Let me update the tests.

Reply:
Done.